docs/piwiks-extensibility-points.md

```diff
@@ -5,189 +5,8 @@ category: CoreDevelop
 
 Plugins can extend Piwik using the following methods:
 
-- using **events**
-- implementing **special classes** that are recognized by Piwik
-- extending certain **abstract base classes**
```

Comment:
Whether we extend something, whether a base class is abstract or whatever is a detail. We link to existing guides now